lib/phases/reboot.js

```diff
@@ -41,6 +42,7 @@ Reboot.prototype.setup = function() {
   this.dispatcher = new Dispatcher(this.device);
   this.registerParser(performanceParser);
   this.registerParser(memoryParser);
+  this.registerParser(filesizeParser);
```

Comment:
Since we only do filesize in a particular test, should we guard this with `if (this.options.remoteFile)`?